_config.yml

```diff
@@ -45,7 +44,13 @@ defaults:
       path: ""
       type: invokers
     values:
-      layout: invoker
+      layout: single
```

Comment:
It's important for invokers to use the invokers layout. We can change the invokers layout to embed within the docs single layout.

Reply:
will reconsider after working on the content first.